tests/Integration/Behaviour/Features/Scenario/Product/update_stock_advanced.feature

```diff
@@ -0,0 +1,292 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags update-stock-advanced
+@reset-database-before-feature
+@clear-cache-before-feature
+@reboot-kernel-before-feature
+@update-stock
+@update-stock-advanced
+Feature: Update product stock from Back Office (BO)
+  As a BO user
+  I need to be able to update product stock from BO
+
+  Background:
+    Given shop "shop1" with name "test_shop" exists
+    # Single shop context is required to modify product quantity
+    And single shop shop1 context is loaded
+    And shop configuration for "PS_ADVANCED_STOCK_MANAGEMENT" is set to 1
+
+  Scenario: I update product stock management
+    Given I add product "product1" with following information:
+      | name       | en-US:Presta camera |
+      | is_virtual | false               |
+    And product "product1" should have following stock information:
+      | use_advanced_stock_management | false       |
+    When I update product "product1" stock with following information:
+      | use_advanced_stock_management | true        |
+    Then product "product1" should have following stock information:
+      | use_advanced_stock_management | true        |
+
+  Scenario: I update product depends on stock (also check automatic update when disabling advanced stock on product)
+    Given I add product "product1" with following information:
```

Comment:
it could be product2  or if its intentionally replaced before each scenario, then it could go to background, no?

Reply:
It could be product2 but then each new scenario will need to always rename the product which is a bit of pain 😅  (and it can lead to mistakes if you forget to replace one) Always using the same reference means it will override the reference to the ID anyway

And I do want to create a new product at each scenario but it's not always "product1" in the pack related scenarios I use "productPack1" for instance So if I always add "product1" in the background in some cases it wouldn't be used